webapp/app.py

```diff
@@ -656,27 +652,10 @@ def cache_headers(response):
     Set cache expiry to 60 seconds for homepage and blog page
     """
 
-    if flask.request.path in ["/core/build"]:
-        response.cache_control.private = True
-
-    if flask.request.path.startswith("/advantage"):
-        response.cache_control.private = True
-        response.headers[
-            "Cache-Control"
-        ] = "no-cache, no-store, must-revalidate"
-        response.headers["Pragma"] = "no-cache"
-
-    if flask.request.path in ["/", "/blog"]:
```

Comment:
This one was to keep right?

Reply:
No, this sets max-age=61. The new default is mas-age=60. I don't think we need to override it any more.